src/components/home/QuoteBox.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const QuoteBox = ({ quote, name }: { quote: string; name: string }) => {
+  return (
+    <div className="relative w-3/5">
+      <div className="absolute right-4 top-4 h-full w-full rounded-xl bg-csa-gray-200 bg-opacity-50"></div>
+      <div className="relative left-0 top-0 flex flex-col gap-8 rounded-xl bg-white p-4">
```

Comment:
```suggestion
      <div className="relative flex flex-col gap-8 rounded-xl bg-white p-4">
```
remove these they shouldn't do anything